day_10/task_1/pipe_maze.cpp

```diff
@@ -2,6 +2,8 @@
 #include <cassert>
 #include <string>
 
+MazePoint invalidMazeEntryPoint(-1, -1);
```

Comment:
Please add constexpr or const to type and change brackets into curly brackets :)

Reply:
done